cmd/clear.go

```diff
@@ -36,14 +36,14 @@ func clearMain(cmd *cobra.Command, args []string) exitCode {
 
 	apiPath, err := client.APIKeyFromPath(path)
 	if err != nil {
-		cmdutil.ErrPrintf("%v\n", err)
+		log.Print(err)
```

Comment:
This is weird to me because `log.Print` is generic but here we're only using it to print errors. Could we handle colored output separately? Or maybe change `log.Print` => `log.PrintError` ? Would prefer separately b/c it seems like there may be some design work there.